### ISSUE TITLE
Add option infix

### ISSUE THREAD

**Issue by smoosies-dev** [acseo/TypesenseBundle#89]
**Describe the solution you'd like**
Add infix option for fields. Perhaps adding more options compared to the Typesense doc would be wise too. Currently I do it with the Configuration.php of the bundle.

**Comment by npotier** [comment]
Hello @smoosies-dev thank you for your suggestion. Could you be more specific about this please ?

> adding more options compared to the Typesense doc

I'll will have a look at the _infix_ suggestion

**Comment by smoosies-dev** [comment]
Yes it is according to the doc and the search possibilities with Typesense. I didn't need any others. infix is ​​specific because it is at the time of indexing that we must say whether we want the field to be there or not, then during the search put always. If I have the case of another I will not fail to inform you.

**Comment by smoosies-dev** [comment]
I found another one that came out. If we want to put fields with the possibility of sorting 'sort_by' we must indicate it with a sort: true on the field but currently it seems to me that this is not possible. So I allowed this setting in the bundle configuration and it works.

**Comment by james2001** [comment]
@smoosies-dev it's comming soon
